diff --git a/node_modules/@vue/compiler-sfc/dist/compiler-sfc.esm-browser.js b/node_modules/@vue/compiler-sfc/dist/compiler-sfc.esm-browser.js
index 3a13d81..e534988 100644
--- a/node_modules/@vue/compiler-sfc/dist/compiler-sfc.esm-browser.js
+++ b/node_modules/@vue/compiler-sfc/dist/compiler-sfc.esm-browser.js
@@ -49205,8 +49205,10 @@ function merge$1(oldMap, newMap) {
   return JSON.parse(mergedMapGenerator.toString())
 }
 
+const nodeRequire = eval('require')
+
 // .scss/.sass processor
-const scss = (source, map, options, load = require) => {
+const scss = (source, map, options, load = nodeRequire) => {
     const nodeSass = load('sass');
     const finalOptions = Object.assign(Object.assign({}, options), { data: getSource(source, options.filename, options.additionalData), file: options.filename, outFile: options.filename, sourceMap: !!map });
     try {
@@ -49228,7 +49230,7 @@ const scss = (source, map, options, load = require) => {
 };
 const sass = (source, map, options, load) => scss(source, map, Object.assign(Object.assign({}, options), { indentedSyntax: true }), load);
 // .less
-const less = (source, map, options, load = require) => {
+const less = (source, map, options, load = nodeRequire) => {
     const nodeLess = load('less');
     let result;
     let error = null;
@@ -49254,7 +49256,7 @@ const less = (source, map, options, load = require) => {
     };
 };
 // .styl
-const styl = (source, map, options, load = require) => {
+const styl = (source, map, options, load = nodeRequire) => {
     const nodeStylus = load('stylus');
     try {
         const ref = nodeStylus(source);
